test/core/Rectangle.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-describe('PIXI.Rectangle', function ()
+describe('PIXI.Rectangle', () =>
 {
-    it('should create a new rectangle', function ()
+    it('should create a new rectangle', () =>
```

Comment:
Could you please revert converting these to fat-arrows? We intentionally kept as `function` so that we could retain the mocha scope, which is more useful here.